samples/compression/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+set(proj_name compression-samples)
+project(${proj_name})
+cmake_minimum_required(VERSION 2.8)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}")
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin")
+
+find_package(VMF)
+
+if(${WIN32})
+    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /WX")
+elseif(${UNIX} AND ${CMAKE_SYSTEM_NAME} MATCHES "Linux")
+    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++0x -Wall")
+endif()
+
+include_directories(${VMF_INCLUDE_DIR})
+link_directories(${VMF_LIB_DIR})
+
+add_executable(compression-builtin compression-builtin.cpp compress_common.cpp)
```

Comment:
let't use either dash (`-`) or underscore (`_`) in file names, not both :)

the underscore is already used in other places, let chose it